src/interpreter/ByteCode.cpp

```diff
@@ -119,7 +119,7 @@ static void clearByteCodeBlock(ByteCodeBlock* self)
 {
 #ifdef ESCARGOT_DEBUGGER
     Debugger* debugger = self->m_codeBlock->context()->debugger();
-    if (debugger != nullptr) {
+    if (debugger != nullptr && self->codeBlock()->markDebugging()) {
```

Comment:
`self->m_codeBlock` vs `self->codeBlock()`
Which one should be preferred?

Reply:
Good catch! I fixed it using `codeBlock()` method